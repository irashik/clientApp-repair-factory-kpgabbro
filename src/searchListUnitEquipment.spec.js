import React from 'react';

import SearchList  from './searchListUnitEquipment';

       


describe("<SearchList/>", () => {
    test('should render correctly empty list', () => {

        const component = shallow(<SearchList />)
        expect(component).toMatchSnapshot();
    });



    



});



