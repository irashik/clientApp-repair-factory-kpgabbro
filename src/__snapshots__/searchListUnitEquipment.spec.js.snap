// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchList/> should render correctly empty list 1`] = `""`;
